components/camel-graphql/src/main/java/org/apache/camel/component/graphql/GraphqlEndpoint.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.camel.component.graphql;
+
+import org.apache.camel.Component;
+import org.apache.camel.Consumer;
+import org.apache.camel.Processor;
+import org.apache.camel.Producer;
+import org.apache.camel.support.DefaultEndpoint;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+public class GraphqlEndpoint extends DefaultEndpoint {
+
+    private final GraphqlConfiguration configuration;
```

Comment:
Here we can add `@UriParam` in order tell Camel's maven plugin that is our configuration that we wish to generate the endpoint property configurer, generate endpoints docs .etc

Reply:
Done